lib/ebae_web/templates/sell/form.html.eex

```diff
@@ -0,0 +1,23 @@
+<%= form_for @changeset, @action, fn f -> %>
+  <%= if @changeset.action do %>
+    <div class="alert alert-danger">
+      <p>Oops, something went wrong! Please check the errors below.</p>
+    </div>
+  <% end %>
+
+  <%= label f, :name %>
+  <%= text_input f, :name %>
+  <%= error_tag f, :name %>
+
+  <%= label f, :description %>
+  <%= text_input f, :description %>
+  <%= error_tag f, :description %>
+
+  <%= label f, :initial_price %>
```

Comment:
could this be a partial <%= text_field, f, key %> or something

Reply:
I like it! Removed a bunch of duplication across the forms.